.gitmodules

```diff
@@ -1,9 +1,6 @@
 [submodule "third_party/testsuite"]
 	path = third_party/testsuite
 	url = https://github.com/WebAssembly/testsuite
-[submodule "third_party/v8/v8"]
-	path = third_party/v8/v8
-	url = https://chromium.googlesource.com/v8/v8.git
 [submodule "third_party/gtest"]
```

Comment:
My understanding is that older versions of MSVC (even 2013, I think?) don't support parsing hexadecimal floats via `strtod`. I implemented this (see `wasm-literal.c`), and wanted to add tests for it. `hexfloat_test` does exhaustive roundtrip testing of floats, and tests 4 billion doubles too. They don't run on Travis because they're kind of slow.